usaspending_api/database_scripts/matviews/drop_old_matviews.sql

```diff
@@ -11,4 +11,5 @@ DROP MATERIALIZED VIEW IF EXISTS summary_view_naics_codes_old;
 DROP MATERIALIZED VIEW IF EXISTS summary_view_old;
 DROP MATERIALIZED VIEW IF EXISTS summary_view_psc_codes_old;
 DROP MATERIALIZED VIEW IF EXISTS universal_award_matview_old;
-DROP MATERIALIZED VIEW IF EXISTS universal_transaction_matview_old;
\ No newline at end of file
+DROP MATERIALIZED VIEW IF EXISTS universal_transaction_matview_old;
+DROP TABLE IF EXISTS universal_transaction_tableview_old;
```

Comment:
I don't see it being used anywhere.  Unfortunately, I am not familiar with this code, however, since my goal was the tableviews to basically be indistinguishable from materialized views, I stuffed it in here.

Reply:
@tony-sappe - Do you know if this script is called? 
@kbard  - I think we should leave this out of the matview script. If you want to build upon the parallels between tableview and matview, create a `usaspending_api/database_scripts/matviews/drop_oldt_tableviews.sql` script to have on deck ... but I think it's better to defer creation of that until we feel we need it.